cpp/example/src/process_csv.cpp

```diff
@@ -0,0 +1,69 @@
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <rmm/mr/device/cuda_memory_resource.hpp>
+#include <rmm/mr/device/per_device_resource.hpp>
+
+#include <cudf/aggregation.hpp>
+#include <cudf/groupby.hpp>
+#include <cudf/io/csv.hpp>
+#include <cudf/table/table.hpp>
+
```

Comment:
I think we should follow `cudf` convention for header inclusion: include from near to far (https://github.com/rapidsai/cudf/blob/branch-0.19/cpp/docs/DEVELOPER_GUIDE.md#includes). So, include the headers from `cudf` first, then `rmm`, and finally from std.